scenarios/web/spring-boot/src/main/resources/templates/layout.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org"
+	xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout">
+<head>
+<meta charset="utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title layout:title-pattern="$CONTENT_TITLE"></title>
+<link rel="stylesheet"
+	th:href="@{/lib/bootstrap/dist/css/bootstrap.min.css}" />
```

Comment:
I discarded some concepts like environment tag helper and script tag helper. They don't have anything like that.